pandas/tests/dtypes/test_common.py

```diff
@@ -324,11 +324,13 @@ def test_is_datetimelike():
 
 @pytest.mark.parametrize(
     "dtype",
-    [pd.Series([1, 2])]
-    + ALL_INT_DTYPES
-    + to_numpy_dtypes(ALL_INT_DTYPES)
-    + ALL_EA_INT_DTYPES
-    + to_ea_dtypes(ALL_EA_INT_DTYPES),
+    [
+        type(pd.Series([1, 2])),
```

Comment:
Is this change to `type` intentional?

Reply:
Yes. If I understand the list correctly then a Series is a data type that should be accounted for in these tests, right?

Reply:
I can see where the naming is confusing but you should keep that the same - `is_integer_dtype` works on types and arrays

Reply:
Alright, I reverted that change.